io.github.mhogomchungu.sirikali.json

```diff
@@ -0,0 +1,61 @@
+{
+    "app-id": "io.github.mhogomchungu.sirikali",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "command": "io.github.mhogomchungu.sirikali",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--filesystem=host",
```

Comment:
you don't explain why you need filesystem host

Reply:
I already did in the "Please describe the application briefly" section above.

Reply:
that doesn't justify it. because to me it looks like you don't need you already escape the sandbox by runing the command outside of it.